oemer/build_system.py

```diff
@@ -237,15 +243,16 @@ def get_track_clef(self):
                 if clef:
                     clefs.append(clef[0])
                 else:
-                    clef = Clef()
-                    clef.track = track
-                    clef.group = self.group
-                    clef.label = ClefType(track%2 + 1)
-                    clefs.append(clef)
+                    new_clef = Clef()
+                    new_clef.track = track
+                    new_clef.group = self.group
+                    new_clef.label = ClefType(track%2 + 1)
+                    clefs.append(new_clef)
             return clefs
-        return [None for _ in range(track_nums)]
+        return [None for _ in range(track_nums)] # type: ignore
 
-    def align_symbols(self):
+    @typing.no_type_check
```

Comment:
Why disable type check here?

Reply:
There are about 20 type errors in this method. With this PR I try to be conservative and to not introduce any code changes. And in the end I didn't feel confident enough to fix the type errors in this function.

Improving this could be something done in later PRs.